modules/polygon.py

```diff
@@ -0,0 +1,63 @@
+import numpy as np
+from numba import jit, njit
```

Comment:
I've not used numba before. Is it effective?

Reply:
Yes! I was surprised how the code optimisation improved when I parallelised the loops with Numba to deal with the polygon functions. For each latitude, I need to call the polygon functions 1440 times (longitude dimension) in ORCA025. This takes 3-4 seconds with Numba compared to 15-20 seconds of other methods, running with 4 VCPUs.